css/src/components/toggle.scss

```diff
@@ -18,7 +18,7 @@ $toggle-circle-selected-background-color: $body-background !default;
 		transition: $input-transition-duration linear;
 		transition-property: background-color, border-color;
 		border: 0.065em solid $toggle-unselected-border-color;
-		border-radius: $radius-rounded;
```

Comment:
I think we should revert these back to rounded, which is a much better name.

Reply:
Check out this thread - https://github.com/microsoft/atlas-design/pull/369#discussion_r826325848